src/validation/validate.js

```diff
@@ -51,3 +51,39 @@ export function validate(
   visit(documentAST, visitWithTypeInfo(typeInfo, visitor));
   return context.getErrors();
 }
+
+export function validateSDL(
+  documentAST: DocumentNode,
+  schemaToExtend?: ?GraphQLSchema,
+  rules?: $ReadOnlyArray<any> = specifiedSDLRules,
```

Comment:
hmmm, I'm not 100% certain whether or not we should allow people to pass in a `TypeInfo` here. I know internally we have a special `TypeInfo` (centered around dealing with pre-open-source GraphQL oddities) that we always pass into `validate`, but I don't believe it's required for the SDL at all.

Reply:
@mjmahone Are you referring to the `rules` field?
To be more explicit I create and used it here so now it looks like:
```js
rules?: $ReadOnlyArray<SDLValidationRule> = specifiedSDLRules,
```

Reply:
Oh I'm OK with the rules field. For `validate` there is an extra optional argument, `typeInfo?: TypeInfo` which I do not believe is required.